src/Elcodi/Plugin/PinterestBundle/Templating/TwigRenderer.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/*
+ * This file is part of the Elcodi package.
+ *
+ * Copyright (c) 2014 Elcodi.com
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Feel free to edit as you please, and have fun.
+ *
+ * @author Marc Morera <yuhu@mmoreram.com>
+ * @author Aldo Chiecchia <zimage@tiscali.it>
+ */
+
+namespace Elcodi\Plugin\PinterestBundle\Templating;
+
+use Elcodi\Component\Plugin\Entity\Plugin;
+use Elcodi\Component\Plugin\Interfaces\EventInterface;
+use Elcodi\Component\Plugin\Templating\Traits\TemplatingTrait;
+
+/**
+ * Class TwigRenderer
+ */
+class TwigRenderer
+{
+    use TemplatingTrait;
+
+    /**
+     * @var Plugin
+     *
+     * Plugin
+     */
+    protected $plugin;
+
+    /**
+     * Set plugin
+     *
+     * @param Plugin $plugin Plugin
+     *
+     * @return $this Self object
+     */
+    public function setPlugin(Plugin $plugin)
+    {
+        $this->plugin = $plugin;
+
+        return $this;
+    }
+
+    /**
+     * Renders import/export buttons
+     *
+     * @param EventInterface $event Event
+     */
+    public function renderJavascript(EventInterface $event)
+    {
+        if (
+            !$this->plugin->isEnabled() ||
```

Comment:
I'd do this check in an upper layer and don't let the plugin do anything while disabled.